catalog/opensearch/catalog-opensearch-source/src/test/groovy/ddf/catalog/source/opensearch/impl/TestOpenSearchParserImplCreateBBox.groovy

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.source.opensearch.impl
+
+import spock.lang.Specification
+
+class TestOpenSearchParserImplCreateBBox extends Specification {
+
+    def 'create BBox from point radius'() {
+        when:
+        final bbox = OpenSearchParserImpl.createBBoxFromPointRadius(lon, lat, radius)
+
+        then:
+        bbox[0] == expectedWest // minX
+        bbox[1] == expectedSouth // minY
+        bbox[2] == expectedEast // maxX
+        bbox[3] == expectedNorth // maxY
+
+        where:
+        lon  | lat  | radius || expectedWest        | expectedSouth      | expectedEast        | expectedNorth
+        1    | 1    | 1      || 0.9999910054140217  | 0.9999910067839408 | 1.0000089945859783  | 1.0000089932160592
+        -122 | 32   | 804672 || -130.54217040367973 | 24.763410847221635 | -113.45782959632025 | 39.236589152778365
+        -122 | 33   | 804672 || -130.63841021419668 | 25.763410847221635 | -113.36158978580332 | 40.236589152778365
+        -211 | -211 | 1      || -180                | -90                | 180                 | -210.99999100678394
+        211  | 211  | 1      || -180                | 210.99999100678394 | 180                 | 90
+    }
+
+    def 'create BBox from polygon'(String[] polyAry, expectedWest, expectedSouth, expectedEast, expectedNorth) {
```

Comment:
❓ Is this test cases parameters required?

Reply:
No they're not. I like it this way, though, because it's easy to add more cases in the future and it matched the first method.